core/src/main/java/com/linecorp/armeria/server/AbstractVirtualHostBuilder.java

```diff
@@ -196,6 +198,17 @@ public B tls(File keyCertChainFile, File keyFile, @Nullable String keyPassword)
         return self();
     }
 
+    /**
+     * Configures SSL or TLS of this {@link VirtualHost} with an auto-generated self-signed certificate.
+     * Note: You should never use this in production but only for a testing purpose.
```

Comment:
Maybe worth putting in all, or many, caps for emphasis

Reply:
Used `<strong>` :-)